cont3xt/vueapp/src/components/pages/AuditHistory.vue

```diff
@@ -0,0 +1,273 @@
+<template>
+  <div class="container-fluid">
+    <!--  search bar form  -->
+    <b-input-group class="flex-grow-1 mr-2">
+      <template #prepend>
+        <b-input-group-text>
+              <span class="fa fa-search fa-fw"/>
+        </b-input-group-text>
+      </template>
+      <b-form-input
+          tabindex="0"
+          ref="search"
+          v-model="filter"
+          debounce="400"
+          placeholder="Search history by indicator, iType, or tags"
+      />
+      <template #append>
+        <b-button
+            tabindex="0"
+            @click="clearSearchTerm"
+            :disabled="!filter"
+            title="Remove the search text">
+          <span class="fa fa-close" />
+        </b-button>
+      </template>
+    </b-input-group>
+    <!--  search bar form  -->
+
+    <!-- time range inputs -->
+    <time-range-input class="mt-1"
+        v-model="timeRangeInfo" :place-holder-tip="timePlaceHolderTip"/>
+    <!-- /time range inputs -->
+
+    <!--  history table  -->
+    <b-table
+        small
+        hover
+        striped
+        show-empty
+        :fields="fields"
+        :items="auditLogs"
+        :sort-by.sync="sortBy"
+        :sort-desc.sync="sortDesc"
+        empty-text="There is no history to show"
+    >
+      <!--   customize column sizes   -->
+      <template #table-colgroup="scope">
+        <col
+            v-for="field in scope.fields"
+            :key="field.key"
+            :style="{ width: field.setWidth }"
+        >
+      </template>
+      <!--   /customize column sizes   -->
+
+      <!--   Button Column   -->
+      <template #cell(buttons)="data">
+        <b-button v-if="getUser && getUser.removeEnabled"
+            @click="deleteLog(data.item._id)"
+            class="btn btn-xs btn-warning"
+            v-b-tooltip.hover="'Delete log'">
```

Comment:
Use the `noninteractive` feature of tooltips so that hovering over the tooltip doesn't keep it activated. This will help when a user wants to click a button above a button that has a tooltip activated.

https://bootstrap-vue.org/docs/components/tooltip#accessibility